### PR TITLE
Fixes issue #977: dropdown alignment is flexible and responsive

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java [hunk 1]
```diff
@@ -43,14 +55,73 @@ public String cssClassName() {
...
     }
 
-    private final IModel<Alignment> alignment;
+    /**
+     * Alignment for the dropdown of a DropDownButton
+     */
+    public static class Alignment implements ICssClassNameProvider {
+        private static final String CLASS_RIGHT = "end";
+        private static final String CLASS_LEFT = "start";
+        public static final Alignment RIGHT = new Alignment(CLASS_RIGHT);
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java]: Do we need `LEFT` for completeness ?

**Comment by martin-g** [comment]
@nvolpini Could you please test this PR and confirm that it fixes #977 ? Thank you!

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java (hunk 1 above)
Comment (solomax) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java]: Actually `RIGHT` was added for backward compatibility :)
`Alignment.right()` and `Alignment.left()` can be used :)

**Comment by nvolpini** [comment]
> @nvolpini Could you please test this PR and confirm that it fixes #977 ? Thank you!

Yes, it did fixed it.
Thank you.

**Comment by martin-g** [comment]
Thank you both!

**Comment by solomax** [comment]
@martin-g can you merge this one to wicket-10 as well? :)

**Comment by martin-g** [comment]
I already did one hour ago - https://github.com/l0rdn1kk0n/wicket-bootstrap/commit/68a3d386b9d15e4c3fba26a60d4bf75e96837b1f.

**Comment by solomax** [comment]
sorry for the noise :(

**Pull request merged**